splunk_logger.go

```diff
@@ -69,7 +69,7 @@ const (
 	// Number of messages allowed to be queued in the channel
 	defaultStreamChannelSize = 4 * defaultPostMessagesBatchSize
 	// Partial log hold duration (if we are not reaching max buffer size)
-	defaultPartialMsgBufferHoldDuration = 5 * time.Second
+	defaultPartialMsgBufferHoldDuration = 100 * time.Millisecond
```

Comment:
Increasing this to 100 will slow down actual processing time in case of partial messages when its not reaching max buffer size, please test its impact.